extension/chrome.manifest

```diff
@@ -82,3 +82,13 @@ locale   personas   uk-UA       locale/uk-UA/
 locale   personas   vi          locale/vi/
 locale   personas   zh-CN       locale/zh-CN/
 locale   personas   zh-TW       locale/zh-TW/
+locale   personas   ar          locale/ar/
+locale   personas   ast-ES      locale/ast-ES/
+locale   personas   de-DE       locale/de-DE/
+locale   personas   el-GR       locale/el-GR/
+locale   personas   eu-ES       locale/eu-ES/
+locale   personas   fi-FI       locale/fi-FI/
+locale   personas   pt-PT       locale/pt-PT/
+locale   personas   sl-SI       locale/sl-SI/
+locale   personas   sr-RS       locale/sr-RS/
+locale   personas   vi-VN       locale/vi-VN/
```

Comment:
Good catch, I told my self "Remember to update `chrome.manifest` when copying the locale files, but forgot anyway :(